front/app/component-library/utils/styleUtils.ts

```diff
@@ -258,6 +258,17 @@ export const stylingConsts = {
   border: `1px solid ${colors.divider}`,
 };
 
+export const getFontWeightCSS = (fontWeight: FontWeight | undefined) => {
```

Comment:
Not sure if `undefined` for `fontWeight` makes sense for a function that gets fontWeight? 😄 Would make it required. And I would also explicitly add case 'normal'.